config/wdio/wdio.conf.js

```diff
@@ -11,6 +11,7 @@ const ip = process.env.WDIO_EXTERNAL_HOST || localIP.address();
 const externalPort = process.env.WDIO_EXTERNAL_PORT || 8080;
 const internalPort = process.env.WDIO_INTERNAL_PORT || 8080;
 const ci = process.env.TRAVIS || process.env.CI;
+const bail = process.env.BAIL || ci;
```

Comment:
nit: lets scope this env var. Something like `WDIO_BAIL`

Reply:
Done: https://github.com/cerner/terra-toolkit/pull/269/commits/7b3ff34158bbe8a01c4dc893a823947e5dad2e66.